Automattic/sensei
# Update "Course Navigation" naming to "Course Theme"

### Changes proposed in this Pull Request

* This updates all references for the project name to **Course Theme**.

### Testing instructions


* Enable the feature flag: `add_filter( 'sensei_feature_flag_course_navigation', '__return_true' );`
* Create a new course.
* In the sidebar, make sure the default theme under "Course Theme" is "WordPress theme".
* Edit an old course (created before this PR), and make sure "WordPress theme" is also the default option there.
* Save a different option, refresh the editor, and make sure it's persisted.


### Screenshot / Video

<img width="307" alt="Screen Shot 2021-11-04 at 17 42 22" src="https://user-images.githubusercontent.com/876340/140418518-0b5f06f9-b670-4bdd-948a-118504809976.png">

<img width="307" alt="Screen Shot 2021-11-04 at 17 43 20" src="https://user-images.githubusercontent.com/876340/140418531-e136c820-4864-40c4-a4ab-acc5f6586fcf.png">
